content/en/docs/rules/style-guide.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Style Guide of Falco Rules
+description: Adopt best practices when writing and contributing Falco rules
+linktitle: Style Guide of Falco Rules
+weight: 90
+---
+
+## Style Guide
+
+Before diving in, read the sections on Falco rules [basics](https://falco.org/docs/rules/basic-elements/) and [condition syntax](https://falco.org/docs/rules/conditions/). Also, check out existing [upstream rules](https://github.com/falcosecurity/rules/blob/main/rules/falco_rules.yaml) for best practices in writing rules.
+
+In addition, the resources under [references/rules](https://falco.org/docs/reference/rules/) provide complementary information. We highly recommend regularly revisiting each guide to stay up-to-date with the latest advancements of Falco.
+
+
+### Naming
+
+Choose a concise title that summarizes the essence of the rule's purpose.
+
+### Description
+
+Aligning with Falco's rules maturity and adoption [framework](https://github.com/falcosecurity/rules/blob/main/proposals/20230605-rules-adoption-management-maturity-framework.md#rules-maturity-framework), it is encouraged to not just include a longer description of what the rule detects, but also to give advice on how to tune this rule and reduce possible noise. If applicable, elaborate on how to correlate the rule with other rules or data sources for incident response. However, keep them concise.
```

Comment:
Same once we have the specific contributing guide or rules maturity checklist doc in the rules repo, we can update the link. Meanwhile link to the proposal.